recipes/modelsearch/recipe.yaml

```diff
@@ -0,0 +1,48 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/prefix-dev/recipe-format/main/schema.json
+schema_version: 1
+
+context:
+  name: modelsearch
+  version: "1.1"
+  python_min: '3.10'
```

Comment:
Since this is equal to the current global pin, you shouldn't set `python_min`

Reply:
fixed.. thank you